<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewVariantGameDialog</class>
 <widget class="QDialog" name="NewVariantGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Variant Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="GroupBox13">
         <property name="title">
          <string>Players</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <property name="bottomMargin">
           <number>11</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="2" column="0">
           <widget class="QLabel" name="TextLabel2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="q4go.qrc">:/images/stone_w16.png</pixmap>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="TextLabel1">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="q4go.qrc">:/images/stone_b16.png</pixmap>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="playerBlackEdit">
            <property name="text">
             <string>Black</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="TextLabel2_2">
            <property name="text">
             <string>Name:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="playerWhiteEdit">
            <property name="text">
             <string>White</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLineEdit" name="playerWhiteRkEdit">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QLineEdit" name="playerBlackRkEdit">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="TextLabel2_3">
            <property name="text">
             <string>Rank:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GroupBox14">
         <property name="title">
          <string>Board shape</string>
         </property>
         <layout class="QGridLayout">
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="topMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <property name="bottomMargin">
           <number>11</number>
          </property>
          <property name="spacing">
           <number>6</number>
          </property>
          <item row="1" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="TextLabel4">
              <property name="text">
               <string>Komi:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KomiSpinBox" name="komiSpin">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>32767</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::StrongFocus</enum>
              </property>
              <property name="acceptDrops">
               <bool>true</bool>
              </property>
              <property name="decimals">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QHBoxLayout">
            <property name="spacing">
             <number>6</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="TextLabel3">
              <property name="text">
               <string>Board size:</string>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QSpinBox" name="xSizeSpin">
              <property name="minimum">
               <number>3</number>
              </property>
              <property name="maximum">
               <number>25</number>
              </property>
              <property name="value">
               <number>19</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>x</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="ySizeSpin">
              <property name="maximumSize">
               <size>
                <width>60</width>
                <height>32767</height>
               </size>
              </property>
              <property name="minimum">
               <number>4</number>
              </property>
              <property name="maximum">
               <number>36</number>
              </property>
              <property name="value">
               <number>19</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Toroidal Go</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QCheckBox" name="hTorusCheckBox">
            <property name="text">
             <string>Horizontally</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="vTorusCheckBox">
            <property name="text">
             <string>Vertically</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="SizeGraphicsView" name="graphicsView">
       <property name="minimumSize">
        <size>
         <width>240</width>
         <height>0</height>
        </size>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::AnchorViewCenter</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <spacer name="Horizontal Spacing2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SizeGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>sizegraphicsview.h</header>
  </customwidget>
  <customwidget>
   <class>KomiSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>komispinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>playerWhiteEdit</tabstop>
  <tabstop>playerBlackEdit</tabstop>
  <tabstop>xSizeSpin</tabstop>
  <tabstop>ySizeSpin</tabstop>
  <tabstop>komiSpin</tabstop>
 </tabstops>
 <resources>
  <include location="q4go.qrc"/>
 </resources>
 <connections/>
</ui>
